### PR TITLE
Create missing data import docs

### PR DESCRIPTION
**Pull request by VadymSachenko**
## PR Description
JIra: https://spryker.atlassian.net/browse/DOC-92
TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/pbc/all/product-information-management/202212.0/base-shop/import-and-export-data/file-details-product-packaging-unit.csv.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  - title: Execution order of data importers in Demo Shop
+    link: docs/scos/dev/data-import/page.version/demo-shop-data-import/execution-order-of-data-importers-in-demo-shop.html
+---
+
+This document describes the `product_packaging_unit.csv` file to configure information about [product packaging units](docs/pbc/all/product-information-management/{{page.version}}/feature-overviews/packaging-units-feature-overview.html) in your Spryker shop.
+
+
+## Import file dependencies
+
+[TODO: add dependencies; If the file has no dependencies, remove the section. If there are two and more import files, use bullet points.]
```
Comment (VadymSachenko) [inline docs/pbc/all/product-information-management/202212.0/base-shop/import-and-export-data/file-details-product-packaging-unit.csv.md:16]: resolve todo

File: docs/pbc/all/customer-relationship-management/202212.0/import-and-export-data/file-details-company.csv.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+[TODO: add dependencies; If the file has no dependencies, remove the section. If there are two and more import files, use bullet points.]
+
+## Import file parameters
+
+| PARAMETER | REQUIRED |  TYPE | REQUIREMENTS OR COMMENTS | DESCRIPTION |
+| --- | --- | --- | --- | --- |
+| key |&check;| string |  | Company key.|
+| name |&check;| string | | Company name.|
+| is_active | &check; | bool | <ul><li>`1`: active</li><li>`0`: inactive</li></ul> | Defines if a company is active. |
+| status |&check;|string|  <!--TODO: add possible statuses if needed; remove the comment otherwise--> |Company status.|
```
Comment (VadymSachenko) [inline docs/pbc/all/customer-relationship-management/202212.0/import-and-export-data/file-details-company.csv.md:24]: resolve todo

File: docs/pbc/all/product-information-management/202212.0/base-shop/import-and-export-data/file-details-configurable-bundle-template.csv.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+related:
+  - title: Execution order of data importers in Demo Shop
+    link: docs/scos/dev/data-import/page.version/demo-shop-data-import/execution-order-of-data-importers-in-demo-shop.html
+---
+
+This document describes the `configurable_bundle_template.csv` file to configure information about [configurable bundle](/docs/pbc/all/product-information-management/{{page.version}}/base-shop/feature-overviews/configurable-bundle-feature-overview.html) templates in your Spryker shop.
+
+## Import file dependencies
+
+[TODO: add dependencies; If the file has no dependencies, remove the section. If there are two and more import files, use bullet points.]
```
Comment (VadymSachenko) [inline docs/pbc/all/product-information-management/202212.0/base-shop/import-and-export-data/file-details-configurable-bundle-template.csv.md:15]: resolve todo

File: docs/pbc/all/product-information-management/202212.0/base-shop/import-and-export-data/file-details-product-list.csv.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+related:
+  - title: Execution order of data importers in Demo Shop
+    link: docs/scos/dev/data-import/page.version/demo-shop-data-import/execution-order-of-data-importers-in-demo-shop.html
+---
+
+This document describes the `product_list.csv` file to configure information about [product lists](https://docs.spryker.com/docs/pbc/all/product-information-management/202212.0/base-shop/feature-overviews/product-lists-feature-overview.html) in your Spryker shop.
+
+## Import file dependencies
+
+[TODO: add dependencies; If the file has no dependencies, remove the section. If there are two and more import files, use bullet points.]
```
Comment (VadymSachenko) [inline docs/pbc/all/product-information-management/202212.0/base-shop/import-and-export-data/file-details-product-list.csv.md:15]: resolve todo

File: docs/pbc/all/product-information-management/202212.0/base-shop/import-and-export-data/file-details-product-measurement-unit.csv.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  - title: Execution order of data importers in Demo Shop
+    link: docs/scos/dev/data-import/page.version/demo-shop-data-import/execution-order-of-data-importers-in-demo-shop.html
+---
+
+This document describes the `product_measurement_unit.csv` file to configure information about [measurement units](/docs/pbc/all/product-information-management/{{page.verison}}/feature-overviews/measurement-units-feature-overview.html) in your Spryker shop.
+
+
+## Import file dependencies
+
+[TODO: add dependencies; If the file has no dependencies, remove the section. If there are two and more import files, use bullet points.]
```
Comment (VadymSachenko) [inline docs/pbc/all/product-information-management/202212.0/base-shop/import-and-export-data/file-details-product-measurement-unit.csv.md:16]: resolve todo

File: docs/pbc/all/request-for-quote/202212.0/import-and-export-data/file-details-quote-request.csv.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+description: The quote_request.csv file to configure information about quote request versions in your Spryker shop.
+template: data-import-template
+last_updated: Jun 1, 2023
+---
+
+This document describes the `quote_request.csv` file to configure information about [quote requests](https://docs.spryker.com/docs/pbc/all/request-for-quote/202212.0/request-for-quote.html) in your Spryker shop.
+
+## Import file dependencies
+
+[TODO: add dependecies; if no dependencies, delete this section]
```
Comment (VadymSachenko) [inline docs/pbc/all/request-for-quote/202212.0/import-and-export-data/file-details-quote-request.csv.md:12]: resolve todo

File: docs/pbc/all/shopping-list-and-wishlist/202204.0/base-shop/import-and-export-data/file-details-shopping-list.csv.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+related:
+  - title: Execution order of data importers in Demo Shop
+    link: docs/scos/dev/data-import/page.version/demo-shop-data-import/execution-order-of-data-importers-in-demo-shop.html
+---
+
+This document describes the `shopping_list.csv` file to configure information about [shopping lists](/docs/pbc/all/shopping-list-and-wishlist/{{page.version}}/base-shop/shopping-lists-feature-overview/shopping-lists-feature-overview.html) in your Spryker shop.
+
+## Import file dependencies
+
+[TODO: add dependencies; If the file has no dependencies, remove the section. If there are two and more import files, use bullet points.]
```
Comment (VadymSachenko) [inline docs/pbc/all/shopping-list-and-wishlist/202204.0/base-shop/import-and-export-data/file-details-shopping-list.csv.md:15]: resolve todo

File: docs/pbc/all/shopping-list-and-wishlist/202212.0/base-shop/import-and-export-data/file-details-shopping-list.csv.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+related:
+  - title: Execution order of data importers in Demo Shop
+    link: docs/scos/dev/data-import/page.version/demo-shop-data-import/execution-order-of-data-importers-in-demo-shop.html
+---
+
+This document describes the `shopping_list.csv` file to configure information about [shopping lists](/docs/pbc/all/shopping-list-and-wishlist/{{page.version}}/base-shop/shopping-lists-feature-overview/shopping-lists-feature-overview.html) in your Spryker shop.
+
+## Import file dependencies
+
+[TODO: add dependencies; If the file has no dependencies, remove the section. If there are two and more import files, use bullet points.]
```
Comment (VadymSachenko) [inline docs/pbc/all/shopping-list-and-wishlist/202212.0/base-shop/import-and-export-data/file-details-shopping-list.csv.md:15]: resolve todo

File: _data/sidebars/pbc_all_sidebar.yml [hunk 9]
```diff
@@ -1819,7 +1826,7 @@ entries:
       - title: Product Information Management
         url: /docs/pbc/all/product-information-management/product-information-management.html
         nested:
-          - title: Base shop
+          - title: Base shop
```
Comment (VadymSachenko) [inline _data/sidebars/pbc_all_sidebar.yml:1829]:
```suggestion
          - title: Base shop
```

File: docs/pbc/all/customer-relationship-management/202212.0/import-and-export-data/file-details-company-business-unit.csv.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+related:
+  - title: Execution order of data importers in Demo Shop
+    link: docs/scos/dev/data-import/page.version/demo-shop-data-import/execution-order-of-data-importers-in-demo-shop.html
+---
+
+This document describes the `company_business_unit.csv` file to configure information about [business units](docs/pbc/all/customer-relationship-management/{{page.version}}/company-account-feature-overview/business-units-overview.html) in your Spryker shop.
+
+## Import file dependencies
+
+[TODO: Add dependencies if any; remove the section otherwise]
```
Comment (VadymSachenko) [inline docs/pbc/all/customer-relationship-management/202212.0/import-and-export-data/file-details-company-business-unit.csv.md:15]: resolve todo

File: docs/pbc/all/customer-relationship-management/202212.0/import-and-export-data/file-details-company-business-unit.csv.md [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+
+## Import file parameters
+
+| PARAMETER | REQUIRED |  TYPE | DESCRIPTION |
```
Comment (VadymSachenko) [inline docs/pbc/all/customer-relationship-management/202212.0/import-and-export-data/file-details-company-business-unit.csv.md:19]: Review the table; make sure that the REQUIRED, TYPE, AND DESCRIPTION fields contain correct information.

File: docs/pbc/all/customer-relationship-management/202212.0/import-and-export-data/file-details-company-user.csv.md [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+related:
+  - title: Execution order of data importers in Demo Shop
+    link: docs/scos/dev/data-import/page.version/demo-shop-data-import/execution-order-of-data-importers-in-demo-shop.html
+---
+
+This document describes the `company_user.csv` file to configure information about [company accounts](/docs/pbc/all/customer-relationship-management/{{page.version}}/company-account-feature-overview/company-accounts-overview.html) in your Spryker shop.
+
+## Import file dependencies
+
+[TODO: Add dependencies if any; remove the section otherwise]
```
Comment (VadymSachenko) [inline docs/pbc/all/customer-relationship-management/202212.0/import-and-export-data/file-details-company-user.csv.md:15]: resolve todo

File: docs/pbc/all/customer-relationship-management/202212.0/import-and-export-data/file-details-company-user.csv.md [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+
+## Import file parameters
+
+| PARAMETER | REQUIRED |  TYPE | REQUIREMENTS OR COMMENTS | DESCRIPTION |
```
Comment (VadymSachenko) [inline docs/pbc/all/customer-relationship-management/202212.0/import-and-export-data/file-details-company-user.csv.md:19]: Review the table; make sure that all fields contain correct information.

File: docs/pbc/all/customer-relationship-management/202212.0/import-and-export-data/file-details-company-business-unit.csv.md [hunk 14]
```diff
@@ -0,0 +1,42 @@
...
+| --- | --- | --- | --- |
+| company_key |&check;| string | Company identifier of the busines unit.|
+| business_unit_key |&check;| string | Business unit identifier.|
```
Comment (AsonUnique) [inline docs/pbc/all/customer-relationship-management/202212.0/import-and-export-data/file-details-company-business-unit.csv.md:22]:
```suggestion
| company_key |&check;| string |  Internal data import identifier of the company.|
| business_unit_key |&check;| string |  Internal data import identifier of the company business unit. |
```

File: docs/pbc/all/customer-relationship-management/202212.0/import-and-export-data/file-details-company-business-unit.csv.md [hunk 15]
```diff
@@ -0,0 +1,42 @@
...
+| name | &check; | string | Business unit name. |
+| email | &check; | string | Business unit email. |
+| phone | &check; | string | Business unit phone number. |
+| external_url | | string | Link to the buisness unit. |
```
Comment (AsonUnique) [inline docs/pbc/all/customer-relationship-management/202212.0/import-and-export-data/file-details-company-business-unit.csv.md:26]: I don't know what this field is used for, it's better to clarify it with the PO department.

File: docs/pbc/all/customer-relationship-management/202212.0/import-and-export-data/file-details-company-business-unit.csv.md [hunk 16]
```diff
@@ -0,0 +1,42 @@
...
+## Import file parameters
+
+| PARAMETER | REQUIRED |  TYPE | DESCRIPTION |
+| --- | - | --- | --- |
+| company_key |&check;| string |  Internal data import identifier of the company.|
+| business_unit_key |&check;| string |  Internal data import identifier of the company business unit. |
+| name | &check; | string | Business unit name. |
+| email | | string | Business unit email. |
+| phone | | string | Business unit phone number. |
+| external_url | | string | Link to the buisness unit. |
```
Comment (VadymSachenko) [inline docs/pbc/all/customer-relationship-management/202212.0/import-and-export-data/file-details-company-business-unit.csv.md:26]:
```suggestion
| external_url | | string | Link to internal or external resources related to the business unit. |
```

**Pull request merged**